tests/model_test.py

```diff
@@ -115,16 +131,16 @@ def test_two_choppers_one_opening():
     assert len(ch2_tofs.blocked) == 1
     assert sc.isclose(
         ch2_tofs.visible.data.coords['tof'][0],
-        (pulse.wavelengths[1] * chopper2.distance * tof.utils.alpha).to(unit='us'),
+        (wavs[1] * chopper2.distance * tof.utils.alpha).to(unit='us'),
     )
     assert sc.isclose(
         ch2_tofs.blocked.data.coords['tof'][0],
-        (pulse.wavelengths[0] * chopper2.distance * tof.utils.alpha).to(unit='us'),
+        (wavs[0] * chopper2.distance * tof.utils.alpha).to(unit='us'),
```

Comment:
Is `alpha` a common symbol for this? I had to look it up.

Reply:
Changed to `m_over_h`.